### PR TITLE
laravel 9 upgrade

### PR DESCRIPTION
**Pull request by AbdullahGhanem**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -25,11 +25,11 @@
     "require": {
         "php": "^7.1.3 || ^8.0",
         "ext-dom": "*",
-        "andrey-helldar/xml-core": "^1.0",
-        "illuminate/database": "^5.6 || ^6.0 || ^7.0 || ^8.0",
-        "illuminate/support": "^5.6 || ^6.0 || ^7.0 || ^8.0",
+        "andrey-helldar/xml-core": "^2.0",
```
Comment (andrey-helldar) [inline composer.json]:
```suggestion
        "dragon-code/xml-core": "^2.1",
```

**Review by andrey-helldar — APPROVED** [review]
Thank you

**Comment by AbdullahGhanem** [comment]
with my pleasure, I Need to tell you that's a great package thanks

**Comment by andrey-helldar** [comment]
Thank you 😊

**Pull request merged**